content/docs/user-guide/troubleshooting.md

```diff
@@ -75,3 +75,19 @@ DVC only supports [symlinked files](/doc/command-reference/add#add-symlink) as
 valid targets for `dvc add`. If the target path is a directory symlink, or if
 the target path contains any intermediate directory symlinks, `dvc add` will
 fail.
+
+## No possible cache types {#cache-types}
+
+You may encounter this error if DVC cannot find a valid
+[file link type](/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache)
+to use when linking data files from cache into your workspace. To resolve the
```

Comment:
In what situations could this happen? Other than when you have an empty string as value for `cache.type` (if that's even possible). Does it depend on what the file system allow?

Reply:
This can happen if your configured cache types are unsupported for your filesystem (like only using reflink on a platform that doesnt support reflinks), or can happen in situations like https://github.com/iterative/dvc/issues/4797 where a platform/filesystem normally supports the configured link type(s) (symlinks in that example) but the actual link operation itself fails due to some (non-DVC related) network or mount misconfiguration.